Examples/Tests/scraping/analysis_rz.py

```diff
@@ -47,7 +47,7 @@
 # remaining particles and scraped particles is equal to the
 # original number of particles
 ts_full = OpenPMDTimeSeries('./diags/diag2/')
-ts_scraping = OpenPMDTimeSeries('./diags/diag3/')
+ts_scraping = OpenPMDTimeSeries('./diags/diag3/particles_at_eb')
```

Comment:
For this test, would you mind setting `diag.intervals = 3`, so that as to test that the `intervals` feature is working as expected?

Reply:
The intervals is tested in the other test case, `embedded_circle`. I'll take that into account when updating the analysis file for that case.